eng/pipelines/eng-common-sync.yml

```diff
@@ -47,6 +47,11 @@ stages:
           displayName: Sync ${{ parameters.DirectoryToSync }} Directory
 
           steps:
+            - template: /eng/common/pipelines/steps/sparse-checkout.yml
```

Comment:
What about the other stage jobs?

Reply:
Do you mean `./templates/steps/sync-directory.yml` ? I can update those too.

Reply:
No I was meeting "CreateSyncPRs" and "VerifyandMergeSyncPrs" in this file. However the sync-directory might also be able to use it.